components/courses/app/models/courses/season.rb

```diff
@@ -7,6 +7,7 @@ class Season < ApplicationRecord
 
     has_many :mentors
     has_many :questions
+    has_many :interviews
```

Comment:
`has_many :interviews, through: :mentors` and that you don't need `season_id` column